app/src/main/java/ryey/easer/skills/operation/intent/IntentOperationData.java

```diff
@@ -217,7 +228,7 @@ public OperationData applyDynamics(SolidDynamicsAssignment dynamicsAssignment) {
                 String type = extra.type;
                 extras.add(new ExtraItem(key, value, type));
             }
-            data.extras = Extras.mayConstruct(extras);
+            intentData.extras = Extras.mayConstruct(extras);
```

Comment:
This line is actually a fix for the fact that extras are broken at the moment. I could split this up into a separate PR if that's appreciated.